includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```
@@ -1875,39 +1869,145 @@ public function get_login_confirmation_settings() {
+			// An empty map is only authoritative once the legacy per-method options are gone.
+			if ( ! empty( $stored ) || ! $this->has_legacy_per_method_locations() ) {
+				return $stored;
+			}
+		}
+
+		if ( $this->has_legacy_location_settings() ) {
+			return self::build_locations_map_from_legacy( $this->stripe_settings );
+		}
+
+		return self::get_default_locations_map();
+	}
```

PHPMD (2.15.0)

[error] The method get_express_checkout_locations_map is not named in camelCase. (undefined)

(CamelCaseMethodName)